spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestTagDDL.java

```diff
@@ -137,11 +136,11 @@ public void testCreateTagUseDefaultConfig() throws NoSuchTableException {
     long snapshotId = table.currentSnapshot().snapshotId();
     String tagName = "t1";
 
-    AssertHelpers.assertThrows(
-        "unknown snapshot",
-        ValidationException.class,
-        "unknown snapshot: -1",
-        () -> sql("ALTER TABLE %s CREATE TAG %s AS OF VERSION %d", tableName, tagName, -1));
+    Assertions.assertThatThrownBy(
+            () -> sql("ALTER TABLE %s CREATE TAG %s AS OF VERSION %d", tableName, tagName, -1))
+        .as("unknown snapshot")
```

Comment:
most (if not all cases) don't need `.as()` because it's implicit either through the expected exception or the error msg.

Reply:
Addressed

Reply:
this one in particular still uses `.as()`

Reply:
Removed